server/routes/key-worker-profile/controller.ts

```diff
@@ -39,74 +33,9 @@ export class KeyWorkerProfileController {
         lastName: prisoner.lastName,
         location: prisoner.cellLocation,
         releaseDate: formatDateConcise(prisoner.releaseDate) || '-',
-        csra: prisoner.csra ? prisoner.csra : '-',
+        csra: prisoner.csra || '-',
```

Comment:
Perhaps these || '-' would work better as a Nunjucks filter `valueOrDefault` kind of thing